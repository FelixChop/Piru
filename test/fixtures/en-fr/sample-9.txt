When the scheme went awry, the culprit tried to save face by concocting a far-fetched alibi.
