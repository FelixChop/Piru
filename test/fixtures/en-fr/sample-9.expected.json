[
  {
    "word": "awry",
    "lemma": "awry",
    "definition": "de travers, qui tourne mal",
    "citation": "When the scheme went awry, the culprit tried to save face by concocting a far-fetched alibi.",
    "to_learn": true
  },
  {
    "word": "culprit",
    "lemma": "culprit",
    "definition": "coupable",
    "citation": "When the scheme went awry, the culprit tried to save face by concocting a far-fetched alibi.",
    "to_learn": true
  },
  {
    "word": "far-fetched",
    "lemma": "far-fetched",
    "definition": "tir√© par les cheveux",
    "citation": "When the scheme went awry, the culprit tried to save face by concocting a far-fetched alibi.",
    "to_learn": true
  }
]
